src/main/java/uk/gov/hmcts/reform/roleassignment/domain/service/deleteroles/DeleteRoleAssignmentOrchestrator.java

```diff
@@ -34,16 +35,19 @@
     private PersistenceService persistenceService;
     private ParseRequestService parseRequestService;
     private ValidationModelService validationModelService;
+    private PersistenceUtil persistenceUtil;
     RequestEntity requestEntity;
     AssignmentRequest assignmentRequest;
     Request request;
 
     public DeleteRoleAssignmentOrchestrator(PersistenceService persistenceService,
                                             ParseRequestService parseRequestService,
-                                            ValidationModelService validationModelService) {
+                                            ValidationModelService validationModelService,
+                                            PersistenceUtil persistenceUtil) {
```

Comment:
why we need persistenceUtil?

Reply:
Hi Dharmendra,
Please see the other comment.
Thanks